Bugfix forcepoint empty get (#40435)
demisto/content: Packs/Forcepoint/Integrations/Forcepoint/Forcepoint_description.md

> ## To set up an instance, you need to add the following:

Reviewer: ```suggestion
## Instance Setup

To set up an instance, add the following.
```

### Server URL
The URL of your Forcepoint instance, without the port number. 

Example: `https://forcepointapi.local`

### Credentials and Password

> Your Credentials (Username) and password when logging in to your Forcepoint instance.

Reviewer: ```suggestion
Your credentials (username) and password for logging in to your Forcepoint instance.
```

> ## Note: Version 8.5.5 or later

Reviewer: ```suggestion
## Important Updates for Version 8.5.5 and Later
```

> Per [Forcepoint's documentation](https://help.forcepoint.com/websec/en-us/on-prem/85/mgmt_api_guide/1E0FA6F7-FDD2-41DE-B3A7-9CDE6B2D348E.html), the API Calls to delete URLs/IPs from a category and to delete the category itself have changed.

Reviewer: ```suggestion
Forcepoint's API calls to delete URLs/IPs from a category and to delete the category itself have changed. See [Forcepoint's documentation](https://help.forcepoint.com/websec/en-us/on-prem/85/mgmt_api_guide/1E0FA6F7-FDD2-41DE-B3A7-9CDE6B2D348E.html).
```
